src/components/Form/index.js

```diff
@@ -40,12 +43,14 @@ Form.propTypes = {
     rows: PropTypes.shape({}).isRequired,
     chapter: PropTypes.shape({}),
     onChange: PropTypes.func.isRequired,
+    componentStyles: PropTypes.shape({stylePropType}),
```

Comment:
rename to `questionStyles`. El `PropTypes` que estás usando no tiene sentido, dejalo como `questionStyles: PropTypes.shape({})`

Reply:
Listo